### PR TITLE
docs: add apisix plug-in tech blog

### PR DESCRIPTION
**Pull request by SylviaBABY**
Changes:

add APISIX plug-in tech blog

### CODE REVIEW

**Review by bzp2010 — APPROVED** [review]

File: website/blog/2021-07-27-limit-rep-plug.md [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+
+## 未来展望
+
+通过拖拽的方式，可以使得使用人员更方便地组合插件来满足不同的场景，以提升 API 网关可扩展能力与运维体验。在实际使用过程中，存在如下可以继续优化的问题：
+
+1. 目前元件的边界判断条件还不够丰富，通过继续完善这些条件，以减少不合理的编排组合；
+2. 当前编排示例不多，提供更多的参考示例可方便开发者学习、供用户使用；
+3. 当前 Apache APISIX 使用了插件定义的 code 进行状态返回（异常则返回状态码，请求终止），可以支持更多 HTTP Response 字段甚至修改插件定义来扩展插件编排能力，如下述插件定义：
+
+```json
```
Comment (liuxiran) [inline website/blog/2021-07-27-limit-rep-plug.md:167]:
```json
local _M = {
  "version": 0.1,
  "priority": 2500,
...
 
}
```

Comment (SylviaBABY) [reply website/blog/2021-07-27-limit-rep-plug.md:167]: update

File: website/blog/2021-07-27-limit-rep-plug.md [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+local _M = {
+  "version": 0.1,
+  "priority": 2500,
+  "type": 'auth',
+  "name": plugin_name,
+  "schema": schema,
+  # 新增的 result 字段，可存储插件运行结果，并传递到下个插件。
```
Comment (liuxiran) [inline website/blog/2021-07-27-limit-rep-plug.md:174]:
```json
"result":  {
    "code": {
      "type": "int"
    }
  }
```

Comment (SylviaBABY) [reply website/blog/2021-07-27-limit-rep-plug.md:174]: done

**Review by liuxiran — APPROVED** [review]

**Pull request merged**